app/domain/invoice/batch_create.rb

```diff
@@ -46,7 +46,11 @@ def call
   end
 
   def create_invoice(recipient)
-    invoice_list.group.invoices.build(attributes(recipient)).save
+    begin
+      invoice_list.group.invoices.build(attributes(recipient)).save
+    rescue NoDonationsPresentError
+      false
+    end
```

Comment:
```suggestion
    invoice_list.group.invoices.build(attributes(recipient)).save
  rescue NoDonationsPresentError
    false
```

Wenn die ganze Methode so mit einem `rescue` gesichert wird, braucht es `begin` und `end` nicht und es ist üblich, das `rescue` auf eine Linie mit dem `def` zu schreiben.